src/test/java/pageObjects/PermissionsTest.java

```
@@ -0,0 +1,99 @@
+package pageObjects;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PermissionsTest extends BaseTest{
+    @Test
+    public void checkIfUserCanAddDevice() throws InterruptedException {
```

do we need to throw an exception?